Jenkinsfile

```diff
@@ -141,69 +178,76 @@ def uploadCli(architectures) {
     for (int i = 0; i < architectures.size(); i++) {
```

Comment:
Referring to line 176 - `uploadGetCliToJfrogReleases()` was removed but is still being used. Since we don't want to use that function we might need to handle the getCli.sh like we do in v2 - Upload it to repo21 under each version and distribute as we do with the executables

Reply:
Well, since the getCli.sh for v1 is not expected to ever change, I suggest that we don't apply this logic for v1.